geomesa-hbase/geomesa-hbase-tools/src/main/resources/org/locationtech/geomesa/hbase/tools/ingest-libjars.list

```diff
@@ -7,6 +7,10 @@ geomesa
 gson
 gt
 hbase
+hbase-client
+hbase-server
+hbase-protocol
+htrace
```

Comment:
Hmm... are these in this PR on purpose?  (I'm wondering a few things got merged together.)